### PR TITLE
fix: transaction pause bounded storage

### PR DESCRIPTION
**Pull request by Roznovjak**
This PR removes `without_storage_info` from the Transaction pause pallet and adds a storage migration.

### CODE REVIEW

File: pallets/transaction-pause/src/migration.rs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		weight.saturating_accrue(T::DbWeight::get().reads(status.len() as u64));
+
+		for ((pallet_name, function_name), _) in status.into_iter() {
+			let pallet_name_b = BoundedVec::<u8, ConstU32<MAX_STR_LENGTH>>::try_from(pallet_name.clone());
+			let function_name_b = BoundedVec::<u8, ConstU32<MAX_STR_LENGTH>>::try_from(function_name.clone());
+			if pallet_name_b.is_err() || function_name_b.is_err() {
+				log::info!(
+					target: TARGET,
+					"Value not migrated because it's too long: {:?}",
+					(pallet_name_b, function_name_b)
```
Comment (apopiak) [inline pallets/transaction-pause/src/migration.rs:59]: probably want to print the unbounded names?

**Review by apopiak — COMMENTED** [review]
hope I'm not being too nitpicky

File: pallets/transaction-pause/src/migration.rs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		StorageVersion::new(1).put::<Pallet<T>>();
+
+		T::DbWeight::get().reads_writes(1, 1)
+	}
+
+	pub fn post_migrate<T: Config>() {
+		assert_eq!(StorageVersion::get::<Pallet<T>>(), 1, "Unexpected storage version.");
+
+		log::info!(target: TARGET, "Transaction pause migration: POST checks successful!");
+	}
```
[the remark this answers is not in the document]
Comment (Roznovjak) [reply pallets/transaction-pause/src/migration.rs]: implemented

File: pallets/transaction-pause/src/migration.rs [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		log::info!(target: TARGET, "Transaction pause migration: PRE checks successful!");
+	}
+
+	pub fn migrate<T: Config>() -> Weight {
+		log::info!(target: TARGET, "Running migration to v1 for Transaction pause");
+
+		let mut weight = Weight::zero();
+
+		let status = v0::PausedTransactions::<T>::drain().collect::<Vec<_>>();
+		weight.saturating_accrue(T::DbWeight::get().reads(status.len() as u64));
```
[the remark this answers is not in the document]
Comment (Roznovjak) [reply pallets/transaction-pause/src/migration.rs]: Couldn't find these numbers in the docs. I'm just curious, how do you know it's 1w/1r? Thanks.

**Pull request merged**